tests/src/whisk/core/controller/test/ActionsApiTests.scala

```diff
@@ -237,6 +244,55 @@ class ActionsApiTests extends ControllerTestCommon with WhiskActionsApi {
         }
     }
 
+    it should "reject create with exec which is too big" in {
+        implicit val tid = transid()
+        val code = "a" * (actionLimit.toBytes / 2L).toInt + 1
```

Comment:
(actionLimit.toBytes / 2L).toInt + 1 -> ((actionLimit.toBytes / 2L).toInt + 1) here and elsewhere.